tests/Unit/Domain/Test_BlockAndElementLogicalCoordinates.cpp

```diff
@@ -117,25 +115,23 @@ void fuzzy_test_block_and_element_logical_coordinates(
           ElementLogicalCoordHolder<Dim>{coords, coord_holder.second.offsets});
     }
     return holders;
-  }
-  ();
+  }();
 
   // Transform element_coords to frame coords
-  const auto frame_coords =
-      [&n_pts, &domain, &element_ids, &element_coords ]() noexcept {
-    tnsr::I<DataVector, Dim, TargetFrame> coords(n_pts);
+  const auto frame_coords = [&n_pts, &domain, &element_ids,
```

Comment:
Probably rename to `inertial_coords` as this `frame` is no longer a general frame.